docs/docs/virtual-hosts/edit-virtual-host.md

```diff
@@ -4,15 +4,15 @@ intro: Edit a Virtual Host to change how an application is accessible on the int
 links:
     overview:
     quickstart:
-    previous: virtual-hosts/view-virtual-host
-    next:
     guides:
     related:
         - virtual-hosts/list-virtual-hosts
         - virtual-hosts/view-virtual-host
     featured:
 required_permissions:
     - virtual-host:update
+    previous:
+    next:
 ---
```

Comment:
Why `previous` and `next` moved under `required_permissions`?

Reply:
removed, although it wasn't exactly under the required permissions property.